dataflowAPI/h/slicing.h

```diff
@@ -261,6 +276,7 @@ class Slicer {
   public:
     typedef std::pair<ParseAPI::Function *, int> StackDepth_t;
     typedef std::stack<StackDepth_t> CallStack_t;
+    DATAFLOW_EXPORT int slicingSizeLimitFactor() { return 100; }
```

Comment:
It has been there for a long time to avoid slicing completely going out of control:

https://github.com/dyninst/dyninst/blob/master/dataflowAPI/src/slicing.C#L298

I think a better strategy is that I set the default value to be -1; a negative value won't bound slicing. And then change existing code to override this function to return 100. 

I have create such interface because for GPU code, there tends to be more instructions, which requires either no bound or a larger bound.

Reply:
And I forgot to make it virtual...